plugins/view-transitions/readme.txt

```diff
@@ -56,6 +56,19 @@ Contributions are always welcome! Learn more about how to get involved in the [C
 
 == Changelog ==
 
+= 1.1.0 =
```

Comment:
Changelog missing for #2038

Reply:
Oh right! Looks like you added the `no milestone` label to it - we shouldn't do that unless we don't want it to be part of a specific milestone.

To avoid this from happening, it's better to either assign a milestone to the PR immediately or leave it off entirely - totally okay to ignore the PR validation failure that would happen then, as it only needs to pass before merge, not immediately.

Just fixed it.